pyartcd/pyartcd/pipelines/update_golang.py

```diff
@@ -40,15 +40,9 @@ def is_latest_build(ocp_version: str, el_v: int, nvr: str, koji_session) -> bool
     if nvr == latest_build[0]['nvr']:
         _LOGGER.info(f'{nvr} is the latest build in {build_tag}')
         return True
-
-    if el_v == 8 and 'module' in nvr:
-        _LOGGER.info(f'{nvr} is not the latest build in build tag {build_tag}. Since it is a module build, '
-                     f'we need to update tag inheritance. Use the flag --create-tagging-ticket to create a jira '
-                     'ticket to do that.')
-    else:
-        override_tag = f'rhaos-{ocp_version}-rhel-{el_v}-override'
-        _LOGGER.info(f'{nvr} is not the latest build in {build_tag}. Run `brew tag {override_tag} {nvr}` to tag the '
-                     f'build and then run `brew regen-repo {build_tag}` to make it available.')
+    override_tag = f'rhaos-{ocp_version}-rhel-{el_v}-override'
```

Comment:
maybe extract this to self.override_tags = { '8': "tag", "9": "tag" } ? or a func would be fine too.
Same for build_tag

Reply:
I would keep it in the function, because el_v version could be different for each call